app/main/routes.py

```diff
@@ -55,7 +63,7 @@ def upload():
                 fund=Config.FUND_NAME,
             )
 
-        success, pre_errors, validation_errors, metadata = post_ingest(
+        success, pre_errors, validation_errors, metadata, error_types = post_ingest(
```

Comment:
Maybe we can rename ```pre_errors``` to ```pre_transformation_errors``` or something similar to match how the variable is named in ```data_requests.py```